src/applications/disability-benefits/526EZ/helpers.jsx

```diff
@@ -414,13 +417,50 @@ export const disabilityOption = ({ diagnosticCode, name, ratingPercentage }) =>
   );
 };
 
+export const PRESTART_STATUSES = {
+  none: 'none',
+  expired: 'expired',
+  created: 'created',
+  retrieved: 'retrieved',
+  renewed: 'renewed',
+  ...commonPrestartStatuses
+};
+
+export const prestartPendingStatuses = new Set([PRESTART_STATUSES.none, PRESTART_STATUSES.expired, PRESTART_STATUSES.pending]);
+
+const successHeading = {
+  created: 'We’ve verified your Intent to File',
+  retrieved: 'We’ve retrieved your existing Intent to File',
+  renewed: 'We’ve verified your Intent to File'
+};
```

Comment:
@peggygannon could you please share your thoughts on these success alert headings?